evap/rewards/models.py

```diff
@@ -20,6 +20,12 @@ class RedemptionEventExpired(Exception):
     pass
 
 
+"""
+All of the following objects handle reward points. As reward points might be connected to monetary transactions,
+these objects may not be altered or deleted after creation.
+"""
+
+
```

Comment:
can you put that above the granting model? that's the most relevant thing because there we're giving people money, and that's also the one with the weird append-only behavior. (also i think editing events is not that bad)